### PR TITLE
Image skeleton thinning

### PR DESCRIPTION
**Pull request by jorenham**
#62 
Based on http://stackoverflow.com/q/19840485/1627479

### CODE REVIEW

File: FingerBlox/app/src/main/java/org/fingerblox/fingerblox/ImageProcessing.java [hunk 1]
```diff
@@ -97,6 +103,14 @@ private Mat detectFeatures(Mat skeleton) {
         return results;
     }
 
+    /**
+     * Match using the ratio test and RANSAC.
+     * Returns the ratio matches and the total keypoints
+     */
+    static double matchFeatures(MatOfKeyPoint keypoints, Mat descriptors) {
+        return 1;
```
Comment (jrauhamaa) [inline FingerBlox/app/src/main/java/org/fingerblox/fingerblox/ImageProcessing.java:111]: What happens here? If this is a placeholder for a future implementation, I would just remove it.

Comment (jorenham) [reply FingerBlox/app/src/main/java/org/fingerblox/fingerblox/ImageProcessing.java:111]: Yea it's a placeholder, was doing several things at once

File: FingerBlox/app/src/main/java/org/fingerblox/fingerblox/ImageProcessing.java [hunk 2]
```diff
@@ -820,10 +845,129 @@ private static Mat getEllipseMask(int width, int height) {
...
             Imgproc.ellipse(ellipseMask, box, new Scalar(255), -1);
         }
         return ellipseMask;
     }
 }
+
+class Thinning {
+    private boolean B[][];
+
+    Mat doJaniThinning(Mat Image) {
```
Comment (jrauhamaa) [inline FingerBlox/app/src/main/java/org/fingerblox/fingerblox/ImageProcessing.java:858]: I'm assuming you copied this code from somewhere. Could you add a comment here pointing at the origin?

**Review by jrauhamaa — CHANGES_REQUESTED** [review]
Left two comments in the code.

**Pull request closed without merging**